src/backend/InvenTree/part/test_api.py

```diff
@@ -3339,3 +3339,27 @@ def test_choices(self):
         response = self.patch(url, {'choices': 'a,b,c,d,e,f,f'}, expected_code=400)
 
         self.assertIn('Choices must be unique', str(response.data['choices']))
+
+
+class PartParameterTests(PartAPITestBase):
+    """Unit test for PartParameter API endpoints."""
+
+    def test_export_data(self):
+        """Test data export functionality for PartParameter objects."""
+        url = reverse('api-part-parameter-list')
+
+        response = self.options(
+            url,
+            data={
+                'export': True,
+                'export_plugin': 'inventree-exporter',
+                'part_detail': True,
+                'template_detail': True,
+            },
+            expected_code=200,
+        )
+
+        fields = response.data['actions']['GET'].keys()
```

Comment:
Would it not be easier to find/understand if this test for `data_exporter` behavior would be in either `data_exporter` or `InvenTree` in a generic form? A refactor of this specific section of Business API might remove coverage from a core mechanism